pakoval/translator-app
# Test/saved translations

test SavedTranslation component
- emit
- existence of the component
- get/set localStorage
- check props
- change props
- set class on the component
